<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageProjectFeatures</class>
 <widget class="QDialog" name="PageProjectFeatures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project features</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>299</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPressureUnit">
      <property name="text">
       <string>Pressure unit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureStatic">
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelTemperatureOn">
      <property name="text">
       <string>Temperature wind on</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureWindOn">
      <property name="minimum">
       <double>-273.149999999999977</double>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelTestId">
      <property name="text">
       <string>Test id</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEditTestId"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelModelId">
      <property name="text">
       <string>Model id</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEditModelId"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelTestPointId">
      <property name="text">
       <string>Test point id</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEditTestPointId"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelMach">
      <property name="text">
       <string>Mach</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelPressureStatic">
      <property name="text">
       <string>Pressure static</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureWindOff">
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelTemperatureUnit">
      <property name="text">
       <string>Temperature unit</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBoxTemperatureUnit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelTemperatureWindOff">
      <property name="text">
       <string>Temperature wind off</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureWindOff">
      <property name="minimum">
       <double>-273.149999999999977</double>
      </property>
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxMach">
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="labelAlpha">
      <property name="text">
       <string>Alpha(°)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxAlpha">
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="labelBeta">
      <property name="text">
       <string>Beta(°)</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxBeta">
      <property name="minimum">
       <double>-180.000000000000000</double>
      </property>
      <property name="maximum">
       <double>360.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureDynamic">
      <property name="maximum">
       <double>100000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBoxPressureUnit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelPressureWindOff">
      <property name="text">
       <string>Pressure wind off</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPressureDynamic">
      <property name="text">
       <string>Pressure dynamic</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PageProjectFeatures</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PageProjectFeatures</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
